recipes/powershell.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Cookbook Name:: windows-hardening
+# Recipe:: powershell
+#
+# Copyright (c) 2016 Joe Gardiner, All Rights Reserved.
+
+# Enabling PowerShell script block logging will record detailed information from the processing of PowerShell commands and scripts
+# Powershell ScriptBlock Logging
+registry_key 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\PowerShell\ScriptBlockLogging' do
```

Comment:
they are fairly new but very helpful, sort of same thing as bash history for linux it enables administrators to see any PowerShell commands that have been run on the box and log into event logs. You can optionally move it into a different log file but I think we should leave it in the default event logs as most people off load those anyway. 
Theres a few whitepapers on it from microsoft, heres the powershell team explain it: https://blogs.msdn.microsoft.com/powershell/2015/06/09/powershell-the-blue-team/

I'm submitting the pull request for windows-baseline today